code/drasil-database/lib/Database/Drasil/ChunkDB.hs

```diff
@@ -4,12 +4,11 @@
 -- Changes to ChunkDB should be reflected in the 'Creating Your Project 
 -- in Drasil' tutorial found on the wiki:
 -- https://github.com/JacquesCarette/Drasil/wiki/Creating-Your-Project-in-Drasil
-
 module Database.Drasil.ChunkDB (
   -- * Types
   -- ** 'ChunkDB'
   -- | Main database type
-  ChunkDB(CDB, symbolTable, termTable, defTable),
```

Comment:
I know these are a pain to maintain, but it is much better to be explicit about what is on the public interface of this module.

Reply:
No worries! I used them all in the `drasil-database` package, but you made me realize that I should've been using the lenses instead of the raw record accessors. I fixed this in 6a6a3532809465ff01cae91c6453522235d777c3